tests/conftest.py

```diff
@@ -176,3 +176,39 @@ def catalog(multi_backend_connection, config) -> AggregatorCollectionCatalog:
         backends=multi_backend_connection,
         config=config
     )
+
+
+@pytest.fixture
+def json_capabilities_no_endpoints():
```

Comment:
I'm not sure these capabilities dictionaries have to be fixtures. 
I think they can just be simple constants, which will make usage a bit less intrusive (because you don't have to inject them through test function arguments).